src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/VedtaksperiodeService.kt

```diff
@@ -56,10 +57,18 @@ class VedtaksperiodeService(
         return forrigeVedtaksperioder.avkortFraOgMed(behandling.revurderFra.minusDays(1))
     }
 
-    fun detFinnesVedtaksperioder(
-        behandlingId: BehandlingId,
-        forrigeBehandlingId: BehandlingId?,
-    ) = finnVedtaksperioder(behandlingId)?.isNotEmpty() == true || finnVedtaksperioder(forrigeBehandlingId)?.isNotEmpty() == true
+    fun detFinnesVedtaksperioderPåForrigeBehandling(saksbehandling: Saksbehandling): Boolean {
+        if (saksbehandling.type == BehandlingType.FØRSTEGANGSBEHANDLING ||
+            finnesTidligereIverksatteVedtak(saksbehandling)
+        ) {
```

Comment:
Kan ev bare sjekke at forrigeBehandlingId er null
```suggestion
        val finnesIkkeTidligereIverksatteVedtak = saksbehandling.forrigeBehandlingId == null
        if (finnesTidligereIverksatteVedtak) {
```

Reply:
Ja.. ser nå at alt dette er unødig.`finnVedtaksperioder()` sjekker allerede for `null`